Exercise_A/index.html

```diff
@@ -8,10 +8,98 @@
     <meta name="description" content="" />
     <meta
       name="viewport"

```

Comment:
Great implementation of the bootstrap page!
I like have you have commented throughout your file.
To make your code more readable please remove some of the extra spaces